src/main/java/seedu/bigpp/ui/UI.java

```diff
@@ -2,6 +2,7 @@
 
 import java.io.PrintStream;
 import java.util.Scanner;
+import java.util.ArrayList;
```

Comment:
Since there is only one valid previous menu for each sub-menu as mentioned before, you will not need the ArrayList anymore, but if a tracking system were to actually be used, I suggest using a stack instead.